fava/core/charts.py

```diff
@@ -165,11 +165,16 @@ def net_worth(self, interval):
         txn = next(transactions, None)
         inventory = Inventory()
 
+        today = datetime.datetime.today().replace(
```

Comment:
This should be a date and if the date is irrelevant, use a constant like `date.min`

Reply:
Whoops, you're right. Done.